gateway/webhook/webhook.go

```diff
@@ -247,6 +247,23 @@ func (bt *batchWebhookTransformerT) batchTransformLoop() {
 				continue
 			}
 
+			queryParams := req.request.URL.Query()
+			var tempBody map[string]interface{}
+			err = json.Unmarshal(body, &tempBody)
+
+			if err != nil {
+				req.done <- webhookErrorRespT{err: response.GetStatus(response.RequestBodyReadFailed)}
+				continue
+			}
+
+			tempBody["query_parameters"] = queryParams
+			body, err = json.Marshal(tempBody)
+
+			if err != nil {
+				req.done <- webhookErrorRespT{err: response.GetStatus(response.RequestBodyReadFailed)}
+				continue
```

Comment:
Error types need to be changed

Reply:
can you suggest what the error types should be changed to?

Reply:
Not sure if we have them already but something more appropriate,
- RequestBodyUnmarshalFailed
- RequestBodyMarshalFailed
or something more generic for this kind of errors.

Reply:
got it. I could find only one more marshalling error. So will update it